hazelcast/src/main/java/com/hazelcast/map/impl/operation/AddIndexOperationFactory.java

```diff
@@ -16,6 +16,7 @@
 
 package com.hazelcast.map.impl.operation;
```

Comment:
Probably a long discussion to decide if it's worth it or not. It clutters the inheritance tree just for one bolierplate method. See `PartitionAwareOperationFactory` but I changed it as suggested since should cause no harm in the modified classes - at least for now.